docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -92,27 +98,17 @@ sudo apt install ziti-edge-tunnel
 
 2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
 2. Set the filemode and owner so that members of group `ziti` will be able to read and delete the token file.
-
-  ```text
-  sudo chown -cR :ziti        /opt/openziti/etc/identities
-  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
-  ```
-
 2. Enable and start the service
 
     ```text
     sudo systemctl enable --now ziti-edge-tunnel.service
     ```
 
-2. The process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
-
-    ```text
-    sudo systemctl restart ziti-edge-tunnel.service
-    ```
+2. [Add an Identity](#adding-an-identity)
```

Comment:
and actually, i just made this comment. probably keep this and remove #2 above (Place a copy of the enrollment JWT....)